pkg/apis/istio/v1alpha2/istiocontrolplane_types.proto

```diff
@@ -552,6 +556,15 @@ message ObjectMeta {
     string namespace = 6;
 }
 
+// GOTYPE: map[string]interface{}
+message TypeMapStringInterface {}
+
+// GOTYPE: interface{}
+message TypeInterface {}
+
+// GOTYPE: *IntOrStringForPB
+message IntOrStringForPB {}
```

Comment:
`TypeIntOrString` to match the other synthetic types?

Reply:
Done.